spec/models/authenticator/ldap_spec.rb

```diff
@@ -94,10 +94,14 @@ def normalize(dn)
   end
 
   before(:each) do
-    allow(MiqLdap).to receive(:new).and_return { FakeLdap.new(user_data) }
```

Comment:
Indeed it's not replaced in most of my previous PRs :confused: But see the bottom of the transpec git message: https://github.com/chrisarcand/manageiq/commit/7027017f55b7d3318e174b600b3ad88e65fba7a7 Weird.